### DISCUSSION TITLE
Welcome to generator-springboot Discussions!

### DISCUSSION THREAD

**Discussion by sivaprasadreddy** [sivaprasadreddy/generator-springboot#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jewel73** [comment]
Hello @sivaprasadreddy , I'd like to inquire about the technology stack you used to create this fantastic repository for Java developers. I'm interested in understanding and building a similar project for practice. Could you please share the technologies you employed or provide any helpful resources, such as blogs or documentation, to aid my understanding of the project? I wanted to know how I could replicate the same type of project.

Thanks in advance.

**Comment by sivaprasadreddy** [comment]
@Jewel73 I have described how this generated is created in this blog post: https://www.sivalabs.in/creating-yeoman-based-springboot-generator/

**Reply by Jewel73** [reply]
@sivaprasadreddy  Thanks,  I would like to request that if you add your blog posts related to your GitHub projects in the README file, it would be helpful for others as well. Thank you!

**Reply by rajadilipkolli** [reply]
@Jewel73 , Can you please create a PR for the same?
